examples/kubernetes/values.yaml

```diff
@@ -18,6 +18,7 @@ proxy:
       cloud.google.com/app-protocols: '{"https":"HTTPS"}'
 
 config:
+  rootDomain: localhost.pomerium.io
```

Comment:
@travisgroth-- I forget, is rootDomain still used? It's used to build the service names right?

Reply:
Yep. Definitely want that.